src/patternfly/components/LabelGroup/label-group.scss

```diff
@@ -0,0 +1,105 @@
+.pf-c-label-group {
+  // label group list
+  --pf-c-label-group__list--MarginBottom: calc(var(--pf-global--spacer--xs) * -1);
+  --pf-c-label-group__list--MarginRight: calc(var(--pf-global--spacer--xs) * -1);
+
+  // category
+  --pf-c-label-group--m-category--PaddingTop: var(--pf-global--spacer--xs);
+  --pf-c-label-group--m-category--PaddingRight: var(--pf-global--spacer--xs);
+  --pf-c-label-group--m-category--PaddingBottom: var(--pf-global--spacer--xs);
+  --pf-c-label-group--m-category--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-label-group--m-category--BorderRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-label-group--m-category--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-label-group--m-category--BorderColor: var(--pf-global--BorderColor--300);
+  --pf-c-label-group--m-category--BackgroundColor: var(--pf-global--BackgroundColor--100);
+
+  // Label in group
+  --pf-c-label-group__label--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-label-group__label--MarginBottom: 0;
```

Comment:
is this neceassary ? i dont know if we'll ever want a marginbottom, but maybe we would ?

Reply:
I could go either way with it. Having it allows us to easily change it under `.pf-m-vertical` as opposed to `.pf-m-vertical pf-c-label-group__label { margin-bottom: ... }`

I'm surprised we don't have one currently for when the chip/label groups wrap under the label.

Reply:
ok lets keep it!